test/app/views/core/CreateAccountModal.spec.coffee

```diff
@@ -601,7 +601,6 @@ describe 'CreateAccountModal Vue Store', ->
       spyOn(api.users, 'signupWithFacebook').and.returnValue(Promise.resolve())
       spyOn(api.users, 'signupWithPassword').and.returnValue(Promise.resolve())
       spyOn(api.trialRequests, 'post').and.returnValue(Promise.resolve())
-      spyOn(application.tracker.legacyTracker.legacyTracker, 'updateTrialRequestData').and.returnValue(Promise.resolve())
```

Comment:
Curious about this change?

Reply:
That snuck in from the segment branch, I've removed that commit so this PR is more clear :)